src/Arriba/Arriba.Server/Program.cs

```diff
@@ -9,14 +9,34 @@
 using System;
 using System.Diagnostics;
 using AspNetHost = Microsoft.Extensions.Hosting.Host;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
 
 namespace Arriba.Server
 {
     internal class Program
     {
+        // Is this considered a static logger? Because it only exists in this context and isn't actually passed into 
+        // other methods
+        private static readonly ActivitySource OpenTelemetryActivitySource = new ActivitySource("Microsoft.Arriba.Server");
+
         private static void Main(string[] args)
         {
             Console.WriteLine("Arriba Local Server\r\n");
+            
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .AddSource("Microsoft.Arriba.Server")
+                .AddConsoleExporter()
+                .Build();
+
+            // I believe this using is supposed to wrap around all of your logic
```

Comment:
Probably Run

Reply:
Sorry can you clarify what this means

Reply:
sorry, i was responding to your comment.  it should likely wrap around the Run() call